source/common/http/codec_helper.h

```diff
@@ -25,6 +34,7 @@ class StreamCallbackHelper {
       return;
     }
 
+    reset_callbacks_started_ = true;
```

Comment:
Ah, I didn't know if there was some need to be called after and figured better safe than sorry.  will collapse.